CHANGELOG.md

```diff
@@ -3,6 +3,25 @@ cfncluster-cookbook CHANGELOG
 
 This file is used to list changes made in each version of the cfncluster cookbook.
 
+1.4.0
+-----
+
+Major new features/updates: 
+
+  - Updated to Amazon Linux 2017.09.1
+  - Updated to Ubuntu 16.04 (server release 20171026.1)
```

Comment:
I would say "applied patches to Ubuntu 16.04".  We didn't update to it; we were already there.